data/gloperate/qml/PipelineView.qml

```diff
@@ -39,6 +41,33 @@ Item
                 page.closed();
             }
         }
+
+        Rectangle
+        {
+            x:      100
+            y:      100
+            width:  200
+            height: 200
+
+            color:        'white'
+            border.color: 'black'
+            border.width: 1
+
+            TextureItem
+            {
+                anchors.fill: parent
+                anchors.margins: 1
+
+                //path: 'DemoPipeline.TextureLoadStage.texture'
+                path: 'DemoPipeline.FramebufferStage1.colorTexture'
```

Comment:
Maybe, I don't know. I haven't thought about using properties outside of the pipeline, but it may work. It would also be ok to leave it like this, the pipeline editor only has to update the path when a connection is drawn between a texture/rendertarget-slot and the preview item. This has to be decided by the UI team when work finally starts on the pipeline editor, unfortunately I don't have the time any more to implement this en detail.